e2e/scripts/jira-client.ts

```
@@ -106,47 +94,85 @@ export class JiraE2EClient {
-      method: 'POST',
-      body: JSON.stringify({
-        name: versionName,
-        project: projectKey,
-        released: false,
-        archived: false,
-      }),
+    // Need to get project ID for creating version
+    const project = await this.client.projects.getProject({ projectIdOrKey: projectKey });
+    const version = await this.client.projectVersions.createVersion({
+      name: versionName,
+      projectId: parseInt(project.id || '0', 10),
```

ESLint

[error] Prefer `Number.parseInt` over `parseInt`.

(unicorn/prefer-number-properties)